library/core/src/intrinsics/mod.rs

```diff
@@ -6,6 +6,9 @@
 //! and <https://github.com/rust-lang/rust/blob/master/compiler/rustc_codegen_llvm/src/intrinsic.rs>,
 //! and for const evaluation in <https://github.com/rust-lang/rust/blob/master/compiler/rustc_const_eval/src/interpret/intrinsics.rs>.
 //!
+//! The functions in this module are implementation details of `core` and should
+//! not be used outside of the standard library.
```

Comment:
Maybe moved this up even further so it is the first paragraph after the "title"?

Reply:
:+1: 
```rust
//! Compiler intrinsics.
//!
//! These are the imports making intrinsics available to Rust code. The actual
//! implementations live in the compiler.  The functions in this module are
//! implementation details of `core` and should not be used outside of the
//! standard library. We generally provide access to intrinsics via stable
//! wrapper functions. Use these instead.
```
something like this?

Reply:
Speaking of which, should I use _should_ or _must_?

I can't think of a reasonable use of an intrinsic outside the standard library, but I'd like to avoid accidentally blocking people.

Reply:
:shrug: must feels a bit too strong, it's totally fine to use these intrinsics we just provide no guarantees about them.